rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -2449,14 +2449,14 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 
 void settings_dialog::refresh_countrybox()
 {
-	const auto vec_countries = countries::get_countries();
-	const auto cur_country = m_emu_settings->GetSetting(emu_settings_type::PSNCountry);
+	const auto& vec_countries = countries::g_countries;
+	const auto& cur_country = m_emu_settings->GetSetting(emu_settings_type::PSNCountry);
 
 	ui->psnCountryBox->clear();
 
 	for (const auto& [cnty, code] : vec_countries)
 	{
-		ui->psnCountryBox->addItem(QString::fromStdString(cnty), QString::fromStdString(code));
+		ui->psnCountryBox->addItem(QString::fromStdString(cnty.data()), QString::fromStdString(code.data()));
```

Comment:
```suggestion
		ui->psnCountryBox->addItem(QString::fromUtf8(cnty.data(), static_cast<int>(cnty.size())), QString::fromUtf8(code.data(), static_cast<int>(code.size())));
```

Forgot to mention this, sorry, but that's how you correctly convert a `std::string_view` to `QString`. Maybe there is a helper for this in the codebase, but I checked briefly and didn't spot one.

Reply:
No, i apologize. It's my inexperience. Thanks for the correction.